Winch/Winch.csproj

```diff
@@ -17,8 +17,9 @@
 	</PropertyGroup>
 
 	<ItemGroup>
+		<PackageReference Include="DredgeGameLibs" Version="1.0.4.1" />
 		<PackageReference Include="Lib.Harmony" Version="2.2.2" />
-		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+		<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
```suggestion
		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Reply:
Hmm dunno if this works like in azure where the PR author can just quickly accept suggestion and commit

Reply:
😁

Reply:
Named that commit badly, my bad.

Reply:
Time to rebase! 😈

Reply:
I definitely, for sure, 100%, absolutely know what that is and how to do it, definitely.